LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
A license file generally indicates how others can use the project. It is probably not required yet really.